runner/app/live/pipelines/comfyui.py

```diff
@@ -26,6 +26,10 @@ class Config:
 
     prompt: Union[str, dict] = DEFAULT_WORKFLOW_JSON
 
+    # NOTE: Dimensions must be maintained with the workflow dimensions and is shared with other pipelines
+    width: int = DEFAULT_WIDTH
+    height: int = DEFAULT_HEIGHT
```

Comment:
Shouldn't we force these to always be the default somewhere? I think we're missing that logic on `api.py` to always make it default/default for the comfy pipeline.

Note that we need it there, and not only here, because we want the decoder to resize+crop the input frames to the right resolution as well.

Reply:
Fixed on https://github.com/livepeer/ai-runner/pull/641/commits/b2805033f64a0f4d19eb3a113d1d57bf5a46026b